powertools-logging/src/test/java/software/amazon/lambda/powertools/logging/internal/LambdaLoggingAspectTest.java

```diff
@@ -172,6 +174,21 @@ void shouldLogServiceNameWhenEnvVarSet() throws IllegalAccessException {
                 .containsEntry("service", "testService");
     }
 
+    @Test
+    void shouldLogxRayTraceIdEnvVarSet() {
+        String xRayTraceId = "1-5759e988-bd862e3fe1be46a994272793";
```

Comment:
There could be a negative test as well? Or modify one of the existing ones to check that its not present?

Reply:
Other tests actually covers the negative scenario. this has -> hasSize(EXPECTED_CONTEXT_SIZE + 1).
few other tests have hasSize(EXPECTED_CONTEXT_SIZE)

Reply:
Aha ok. makes sense then 👍